### DISCUSSION TITLE
MicroPython equivalent of this library

### DISCUSSION THREAD

**Discussion by algorni** [jrowberg/i2cdevlib#654] [Q&A]
Hi there, I'm interested in using the MP6050 in a raspi pico project and I would like to leverage MicroPython as a programming language.

Is there any port of this library capable to get the Yaw from an MP6050? 

Thanks
Alberto

**Comment by jrowberg** [comment]
Hi Alberto,

Unfortunately I'm not aware of any MicroPython ports of I2Cdevlib or the MPU6050 code that I wrote. Someone may have done it, or even implemented something from scratch (maybe more likely), but I'm not sure.

A quick search shows this:

https://microcontrollerslab.com/micropython-mpu-6050-esp32-esp8266/

It looks like it's only raw data though, not the DMP. You'd have to implement a 6-axis fusion algorithm on the Pico if you use that.

Jeff
